daemon/api.go

```diff
@@ -1593,7 +1593,7 @@ func splitQS(qs string) []string {
 
 func getSnapConf(c *Command, r *http.Request, user *auth.UserState) Response {
 	vars := muxVars(r)
-	snapName := vars["name"]
+	snapName := systemCoreSnapDealias(vars["name"])
```

Comment:
Nice! Though personally I would say "unalias" or "resolve" instead.

Reply:
I like resolve

Reply:
There's a second part of this work that introduces `system` to interfaces. I introduce a `systemCoreSnapAlias` there that translates `core` -> `system` where needed. If you don't mind I'd prefer to go with `systemCoreSnapUnalias` instead of `*Resolve`, just to keep the symmetry between the two.